app/views/content/funding-and-support/if-youre-disabled.md

```diff
@@ -36,17 +36,19 @@ If you’re disabled, have a mental health condition or educational needs, you c
 
 $international-content$
 
-## Disabled Students’ Allowance and other support
+## Financial support
+
+### Disabled Students' Allowance (DSA)
 
 You can [apply for Disabled Students’ Allowance (DSA)](https://www.gov.uk/disabled-students-allowance-dsa/how-to-claim) to cover the study-related costs you have because of a mental health problem, long term illness or any other disability.
 
 You’ll get confirmation of whether your application has been successful within 6 weeks.
 
 It can take up to 14 weeks to get your DSA support, so you should take this into consideration if you need the support in place for the start of your course.
 
-[If you do paid training you may also be eligible for support](https://www.gov.uk/access-to-work).
+### Other support for funding your training
 
-[AbilityNet](https://abilitynet.org.uk/about-abilitynet) or [Disability Rights UK](https://www.disabilityrightsuk.org/) may also be able to help you identify technology or other adjustments to help you study.
+If you get paid for your training (for example, salaried training or a teacher degree apprenticeship), you may be [eligible for support through Access to Work](https://www.gov.uk/access-to-work).
```

Comment:
This is much better now - just wanted to check we've confirmed that salaried training and TDAs meet the eligibility criteria @emmaachesongray ?

Reply:
Sarah and I discussed it and feel it would based on the gov.uk eligibility criteria: A paid job can be full or part-time and can include:

employment
self-employment
an apprenticeship
a work trial or work experience
an internship
a work placement
You cannot get Access to Work support for voluntary work.

Reply:
Or is there a team we can double check with?